MoltenVK/MoltenVK/GPUObjects/MVKDevice.h

```diff
@@ -830,9 +839,9 @@ class MVKDevice : public MVKDispatchableVulkanAPIObject {
 	void makeResident(id allocation) {}
 #else
 	void makeResident(id<MTLAllocation> allocation) {
```

Comment:
Since we're moving the residency set to `MVKPhysicalDevice`, these functions will be more consistent if moved there too.

Then elsewhere, just call `getPhysicalDevice()->makeResident()` where you now call `getDevice()->makeResident()`.